doc/man3/OSSL_STORE_open.pod

```diff
@@ -98,13 +98,14 @@ B<OSSL_STORE_CTX>, as well as the B<OSSL_STORE_CTX> itself.
 =head1 SUPPORTED SCHEMES
 
 The basic supported scheme is B<file:>.
-Any other scheme can be added dynamically, using
-OSSL_STORE_register_loader().
+Any other scheme can be added dynamically, using OSSL_STORE_register_loader().
 
 =head1 NOTES
 
 A string without a scheme prefix (that is, a non-URI string) is
 implicitly interpreted as using the F<file:> scheme.
+An empty string ("") is interpreted as reference to B<stin>,
+such that input is read, for instance, from the command line or terminal.
```

Comment:
This has all the signs of an ugly hack.  Hold on, I'll extract the `OSSL_STORE_attach()` function from #7390

Reply:
This PR meanwhile is prepared for using `OSSL_STORE_attach()`.

Reply:
It turns out that my hack was not strictly needed, so I've reverted it.